### PR TITLE
Staging/adxl313 fix

### PR DESCRIPTION
**Pull request by danmois**
Fix to the calibbias and offset read/write functionalities for the ADXL31x driver. Behavior is now similar to the one of the Linux driver.

### CODE REVIEW

**Review by amiclaus — APPROVED** [review]
lgtm, just please remove `..` from 68af13e11151cce66cc587da15e8fc8bee454b62 commit title.

File: drivers/accel/adxl313/adxl313.c [hunk 1]
```diff
@@ -488,29 +488,184 @@ int adxl313_get_op_mode(struct adxl313_dev *dev,
...
 	int ret;
-	uint8_t reg_val = 0x00;
+	int32_t raw_offset = 0x00;
+
+	if (axis > ADXL313_Z_AXIS)
+		return -EINVAL;
+
+	/* Read data to register for getting offset register value. */
+	ret = adxl313_get_raw_offset(dev, &raw_offset, axis);
+	if (!ret) {
```
Comment (amiclaus) [inline drivers/accel/adxl313/adxl313.c:556]: please use `if (ret)` and `return ret;` here.

**Comment by danmois** [comment]
V1
- implemented review observations

File: drivers/accel/adxl313/adxl313.c (hunk 1 above)
Comment (danmois) [reply drivers/accel/adxl313/adxl313.c]: Thank you for the observation! I updated the code.

**Pull request merged**